run_server.sh

```diff
@@ -53,6 +53,8 @@ elif [[ $ENV == "feedingamerica" ]]; then
   export FLASK_ENV=local-feedingamerica
 elif [[ $ENV == "stanford" ]]; then
   export FLASK_ENV=local-stanford
+elif [[ $ENV == "stanford-staging" ]]; then
+  export FLASK_ENV=local-stanford-staging
```

Comment:
fyi - this file shouldn't need to be updated with each deployment (it's only used for local development, so we would test the local-stanford instance, then deploy to staging. the `-e` arg also takes an arbitrary string if needed

Reply:
Ok got it, makes sense